tests/unit/source_code/samples/values_across_notebooks_dbutils_notebook_run.py

```diff
@@ -0,0 +1,4 @@
+# Databricks notebook source
+
+dbutils.notebook.run("./values_across_notebooks_child.py")
+spark.table(f"{a}")
```

Comment:
are you sure this is the right logic? my hunch it does not compute on workspace

Reply:
same here